cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -154,7 +154,11 @@ function wait-for-jobs() {
     local rv=0
     local pid
     for pid in $(jobs -p); do
-        wait ${pid} || (rv=1; log ERR "error in pid ${pid}")
```

Comment:
I think if you used curly braces instead of parentheses it would do what you wanted:

```
wait "${pid}" || {
  rv=1
  log ERR "error in pid ${pid}"
}
```